var/spack/repos/builtin/packages/mercury/package.py

```diff
@@ -22,6 +22,11 @@ class Mercury(CMakePackage):
     variant('selfforward', default=True,
             description='Mercury will short-circuit operations' +
                         ' by forwarding to itself when possible')
+# NOTE: the 'udreg' variant requires that the MPICH_GNI_NDREG_ENTRIES=1024
+#   environment variable be set at run time to avoid conflicts with 
```

Comment:
Flake is complaining about trailing whitespace here

Reply:
Whoops, thanks for catching that @scheibelp .  Updated to address.